src/DssSpell.sol

```diff
@@ -18,49 +18,258 @@ pragma solidity 0.6.12;
+            INPUT_CONDUIT: 0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f,
+            OUTPUT_CONDUIT: 0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f,
+            URN:  0xeF1699548717aa4Cf47aD738316280b56814C821,
+            gemID: "RWA004",
+            joinID: "MCD_JOIN_RWA004_A",
+            urnID: "RWA004_A_URN",
+            inputConduitID: "RWA004_A_INPUT_CONDUIT",
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilk_string: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_815_730 * WAD,
+            MAT: 11_000,
+            TAU: 0,
+            DOC: "QmYR2PXwLpdXS8Vp1yS39SPFT1XhmgbsK6XvZ9ApRpNV8M"
+        });
+
+        // https://vote.makerdao.com/polling/QmZCPDVY#poll-detail
+        CentrifugeCollateralValues memory RWA005 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xA4fD373b93aD8e054970A3d6cd4Fd4C31D08192e,
+            GEM: 0x6DB236515E90fC831D146f5829407746EDdc5296,
+            OPERATOR: 0x5b702e1fEF3F556cbe219eE697D7f170A236cc66,
+            INPUT_CONDUIT: 0x5b702e1fEF3F556cbe219eE697D7f170A236cc66,
+            OUTPUT_CONDUIT: 0x5b702e1fEF3F556cbe219eE697D7f170A236cc66,
+            URN:  0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90,
+            gemID: "RWA005",
+            joinID: "MCD_JOIN_RWA005_A",
+            urnID: "RWA005_A_URN",
+            inputConduitID: "RWA005_A_INPUT_CONDUIT",
+            outputConduitID: "RWA005_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA005",
+            ilk: "RWA005-A",
+            ilk_string: "RWA005-A",
+            ilkRegistryName: "RWA005-A: Centrifuge: Fortunafi",
+            RATE: FOUR_PT_FIVE_PCT,
+            CEIL: 15 * MILLION,
+            PRICE: 17_199_394 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmbgDoPn6UcfSDENDqHLgatMFoqXikC8E8it9WaZXyLXmc"
+        });
+
+        // https://vote.makerdao.com/polling/QmdHV2tW#poll-detail
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x5E11E34b6745FeBa9449Ae53c185413d6EdC66BE,
+            GEM: 0x4EE03cfBF6E784c462839f5954d60f7C2B60b113,
+            OPERATOR: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            INPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            OUTPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            URN:  0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilk_string: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
```

Comment:
This might be an appropriate executive cycle to update the Ilk Registry names for `RWA001` and `RWA002`

Reply:
We didn't do it in the `kovan` spell.  We should probably keep a list of housekeeping actions and make a spell with just those.

Reply:
I'd prefer to put those into a smaller spell personally.